manifests/k8s/config/exporter/kustomization.yaml

```diff
@@ -24,6 +24,8 @@ patchesStrategicMerge: []
 # - ./patch/patch-qat.yaml
 # add this line for dcgmi patch
 # - ./patch/patch-dcgmi.yaml
+# add this line for habana patch
+# - ./patch/patch-habana.yaml
```

Comment:
Where is the `patch-habana.yaml` file? I don't see any added files under `manifests/k8s/config/exporter/patch` or maybe I'm just missing something

Reply:
added